.ci/azure-pipelines.yml

```diff
@@ -35,6 +35,13 @@ jobs:
         script: 'git submodule update --recursive --remote'
         workingDirectory: '$(Build.SourcesDirectory)'
 
+    - task: DotNetCoreCLI@2
+      displayName: "Build Server Project in Submodule"
+      inputs:
+        command: build
+        arguments: '--configuration Release --source https://api.nuget.org/v3/index.json'
+        workingDirectory: '$(Build.SourcesDirectory)/src/jellyfin'
+
```

Comment:
This resolves an issue with external references not being resolved correctly in triple-slash comments when running the `docfx` command:  https://github.com/dotnet/docfx/issues/5112#issuecomment-568178162